src/components/Tabs/Tabs.module.scss

```diff
@@ -0,0 +1,16 @@
+.tabButton {
+  background: white;
```

Comment:
```suggestion
  background-color: white;
```

Reply:
both are correct

Reply:
yes but you are using only one value of `background` then it is better to use `background-color`. it also improve the readability and performance. because `background` is a shorthand property to use for multiple value.